ta/pkcs11/src/processing.c

```diff
@@ -625,8 +625,13 @@ enum pkcs11_rc entry_processing_key(struct pkcs11_client *client,
 		 * specified with C_Derive/Unwrap() in the specification. So
 		 * return the next most appropriate error.
 		 */
-		if (rc == PKCS11_CKR_KEY_FUNCTION_NOT_PERMITTED)
```

Comment:
There are already similar fixups in actual implementation commit so why not have these already there?

Reply:
Sure, I will squash these with wrap/unwrap commits.